db/view/view.hh

```diff
@@ -123,6 +123,70 @@ bool matches_view_filter(const schema& base, const view_info& view, const partit
 
 bool clustering_prefix_matches(const schema& base, const partition_key& key, const clustering_key_prefix& ck);
 
+/**
+ * The liveness of rows resulting from an update to a table with an index over collection column
+ * depends on the liveness of the collection cells. bytes_with_action is used to return this information
+ * from the function computing the result for a particular computed column.
+ */
+struct bytes_with_action {
+    struct no_action {};
+    struct shadowable_tombstone_tag {
+        api::timestamp_type ts;
+        shadowable_tombstone into_shadowable_tombstone(gc_clock::time_point now) const {
+            return shadowable_tombstone{ts, now};
+        }
+    };
+
+    struct apply_visitor {
+        deletable_row* row;
+        gc_clock::time_point now;
+        bool apply_row_marker;
+        bool apply_tombstone;
```

Comment:
nitpick: these members have exactly the same name as public methods of another class, which is mildly confusing - perhaps add a `should_` prefix?

Reply:
Makes sense.